src/main/java/org/mskcc/limsrest/util/Utils.java

```diff
@@ -42,6 +42,7 @@
     private final static List<String> CAPTURE_SAMPLE_TYPES = Collections.singletonList("capture library");
     private final static List<String> POOLED_SAMPLE_TYPES = Collections.singletonList("pooled library");
     private final static String FAILED_STATUS_TEXT = "failed";
+    private final static String SEQUENCING_ANALYSIS_STATUS = "Illumina Sequencing Analysis";
```

Comment:
Can you use `org.mskcc.limsrest.util.StatusTrackerConfig.STAGE_SEQUENCING_ANALYSIS`?